.env.example

```
@@ -26,4 +26,12 @@ DOC_PATH=./my-docs
 # Default: 0.0 (no rate limiting)
-#SCRAPER_RATE_LIMIT_DELAY=0.0
\ No newline at end of file
+#SCRAPER_RATE_LIMIT_DELAY=0.0
+
+# LangChain Tracing (LangSmith) - Enable for enhanced observability
+# To enable tracing, uncomment the following lines and provide your API Key
+
+# LANGCHAIN_TRACING_V2=true
+# LANGCHAIN_ENDPOINT="https://api.smith.langchain.com"
+# LANGCHAIN_API_KEY=
+# LANGCHAIN_PROJECT="gpt-researcher"
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)